angular/package.json

```diff
@@ -54,22 +54,22 @@
     "@babel/preset-react": "^7.7.4",
     "@babel/traverse": "^7.6.0",
     "@compodoc/compodoc": "^1.1.11",
-    "@storybook/addon-a11y": "^5.3.18",
-    "@storybook/addon-docs": "^5.3.18",
-    "@storybook/addon-google-analytics": "^5.3.18",
-    "@storybook/addon-jest": "^5.3.18",
-    "@storybook/addons": "^5.3.18",
-    "@storybook/angular": "^5.3.18",
-    "@storybook/preset-scss": "^1.0.2",
-    "@storybook/theming": "^5.3.18",
+    "@storybook/addon-a11y": "^6.3.0",
+    "@storybook/addon-docs": "^6.3.0",
+    "@storybook/addon-google-analytics": "^6.2.9",
+    "@storybook/addon-jest": "^6.3.0",
+    "@storybook/addons": "^6.3.0",
+    "@storybook/angular": "^6.3.0",
+    "@storybook/preset-scss": "^1.0.3",
```

Comment:
This is the latest version for this even though it doesn't match the other pkg versions.